components/SearchBar/index.js

```diff
@@ -130,21 +131,21 @@ function SearchBar({ search, onSearch }) {
               />
             </Styled.FilterItem>
             <Styled.FilterItem>
-              <label>X</label>
+              <label>clear</label>
```

Comment:
Your label should be associated with an input. If there is no input (or output) then it probably should not be a label. 

I looked at the other elements in the filter, and the labels don't have `htmlFor`. It is important we add this for accessibility reasons. 

See [here](https://react.dev/reference/react-dom/components/common#common-props) and [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/for#usage).